### ISSUE TITLE
Sample can't be deployed on assistant

### ISSUE THREAD

**Issue by rayarub** [actions-on-google/dialogflow-digital-goods-nodejs#1]
Attempting to deploy this sample into Alpha on Actions on Google results in an error:

> Intent 'actions.intent.DIGITAL_PURCHASE_CHECK' is using the reserved namespace. Please choose a name that does not start with 'actions.'

Appears to be related to the presence of the "Digital Purchase Check" intent and the "actions_intent_DIGITAL_PURCHASE_CHECK" event on that.

Are there any changes that need to be made to make this code usable?

**Comment by Fleker** [comment]
In Dialogflow, you can give the intent any name you want, as long as that string is the Event name.

**Comment by rayarub** [comment]
Yeah I understand – in this case I'm just deploying the sample agent from this repository, so I'm using the intents as you have them named. So you've got an intent named "Digital Purchase Check" listening for the actions_intent_DIGITAL_PURCHASE_CHECK event. This will work initially, but when you try to make an Alpha release, it produces the error I mentioned in the issue.

So generally, it looks like AoG won't allow the release of a Dialogflow agent that is listening for actions_intent_DIGITAL_PURCHASE_CHECK.

(Sorry, there was a typo originally in the issue that incorrectly referenced COMPLETE_PURCHASE - I've corrected it now).
